ga4gh/cli.py

```diff
@@ -797,11 +797,14 @@ class SearchExpressionLevelsRunner(AbstractSearchRunner):
     def __init__(self, args):
         super(SearchExpressionLevelsRunner, self).__init__(args)
         self._rnaQuantificationId = args.rnaQuantificationId
+        if len(args.feature_ids) > 0:
```

Comment:
what happens if `len(args.feature_ids) <= 0`?  will it error in `run` because `self._featureIds` does not exist?